### PR TITLE
refactor(sbb-timetable-row): enhance a11y and design

### PR DESCRIPTION
**Pull request by osminaz**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

-

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [ ] Chrome Desktop
- [ ] Edge Desktop
- [ ] Safari Desktop
- [ ] Chrome Mobile
- [ ] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/components/sbb-pearl-chain-time/sbb-pearl-chain-time.scss [hunk 1]
```diff
@@ -18,11 +32,12 @@
...
 
 .sbb-pearl-chain__time-walktime--left {
-  margin-inline-end: var(--sbb-spacing-fixed-2x);
+  transform: translateX(sbb.px-to-rem-build(-7));
+  margin-inline-end: calc(var(--sbb-spacing-fixed-2x) - sbb.px-to-rem-build(7));
 }
 
 .sbb-pearl-chain__time-walktime--right {
-  margin-inline-start: var(--sbb-spacing-fixed-2x);
+  margin-inline-start: calc(var(--sbb-spacing-fixed-2x) - 4px);
```
Comment (jeripeierSBB) [inline src/components/sbb-pearl-chain-time/sbb-pearl-chain-time.scss:40]:
```suggestion
  margin-inline-start: calc(var(--sbb-spacing-fixed-2x) - sbb.px-to-rem-build(4));
```

File: src/components/sbb-timetable-row/sbb-timetable-row.helper.tsx [hunk 2]
```diff
@@ -1,32 +1,25 @@
...
 import { i18nDurationMinute, i18nDurationHour } from '../../global/i18n';
 import { PTRideLeg } from '../../global/interfaces/pearl-chain-properties';
-import { Notice, PtSituation, Trip, VehicleModeEnum } from './sbb-timetable-row.custom';
+import { HimCus, Notice, PtSituation, Trip, VehicleModeEnum } from './sbb-timetable-row.custom';
 
 export const durationToTime = (duration: number): string => {
   const result = [];
-  const now = 0;
-  let future = addMinutes(now, duration);
+  const now = new Date();
```
Comment (jeripeierSBB) [inline src/components/sbb-timetable-row/sbb-timetable-row.helper.tsx:10]: For me it doesn't make sense to start from the current date. Even if the output is the same, it's always a different calculation. Do I miss something? My alarm bells always ring if there is a "new Date()" in the code somewhere ;)

File: src/components/sbb-timetable-row/sbb-timetable-row.tsx [hunk 3]
```diff
@@ -219,18 +240,19 @@ export class SbbTimetableRow {
...
+                  noticeAttributes?.map(
                     (notice, index) =>
                       index < 4 && (
                         <li>
                           <sbb-icon
                             class="sbb-travel-hints__item"
-                            name={notice}
-                            aria-hidden="false"
+                            name={'sa-' + notice.name.toLowerCase()}
+                            aria-hidden="true"
```
Comment (jeripeierSBB) [inline src/components/sbb-timetable-row/sbb-timetable-row.tsx:253]:
```suggestion
```
You can remove aria-hidden=true as it is the default of the sbb-icon

File: .storybook/main.js [hunk 4]
```diff
@@ -10,4 +10,8 @@ module.exports = {
   features: {
     postcss: false,
   },
+  managerWebpack: (config, options) => {
```
Comment (jeripeierSBB) [inline .storybook/main.js:13]: what does this, and is this really needed?

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]

File: .storybook/main.js (hunk 4 above)
Comment (osminaz) [reply .storybook/main.js]: I accidentally pushed this one, sorry.

this is our workaround against crashing storybook.

https://github.com/storybookjs/storybook/issues/13795

File: src/components/sbb-timetable-row/sbb-timetable-row.helper.tsx (hunk 2 above)
Comment (osminaz) [reply src/components/sbb-timetable-row/sbb-timetable-row.helper.tsx]: I thought it was a good idea to refactor out the let variable. But i reverted the change.

Comment (jeripeierSBB) [reply src/components/sbb-timetable-row/sbb-timetable-row.helper.tsx]: I think you can also do it with your logic, but set now to 0 anyways ;) As you prefer.

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**